src/LibraryManager.Vsix/UI/Models/SearchTextBoxViewModel.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Web.LibraryManager.Contracts;
+using Microsoft.Web.LibraryManager.Vsix.Search;
+using Microsoft.Web.LibraryManager.Vsix.UI.Controls;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI.Models
+{
+    internal abstract class SearchTextBoxViewModel : BindableBase
```

Comment:
>SearchTextBoxViewModel [](start = 28, length = 22)

tests?

Reply:
Covered by PackageIdViewModelTests

---
In reply to: [289995462](https://github.com/aspnet/LibraryManager/pull/483#discussion_r289995462) [](ancestors = 289995462)